package.json

```diff
@@ -24,6 +24,7 @@
     "@justeat/f-header": "1.1.0",
     "@justeat/f-icons": "1.16.0",
     "@justeat/f-toggle": "1.1.0",
+    "@justeat/f-utilities": "0.5.0",
     "@justeat/f-validate": "1.1.0",
     "@justeat/fozzie": "1.19.0",
```

Comment:
It might be good to update fozzie to `v1.30.1` once [my PR](https://github.com/justeat/fozzie/pull/205) is merged, seeing as they're related.

Reply:
@xander-marjoram I will do it in my PR as I have a small fix for f-utilities component